sbi.adoc

```diff
@@ -1,3 +1,32 @@
 [[sbi]]
 == SBI Requirements
 
+The supervisor binary interface (SBI) cite: [SBI] specification defines the interface
+between the supervisor mode and the next higher privilege mode. This section
+defines the mandatory SBI version and extensions implemented by the higher
+privilege mode in order to be compatible with this specification.
+
+=== Minimum SBI version
+The SBI implementation must be v1.0 or higher. 
+
+=== Mandatory SBI extensions
+
+The SBI v1.0 allows optional SBI extensions. However, the following optional
+extensions are mandatory for a compliant system. 
+
+* SBI Hart State Management (HSM)
+
+* SBI System Reset (SRST)
+
+* SBI Performance Monitoring (PMU)
+
+In addition to this, the following SBI extensions are only mandatory if required
+ISA extension is not available in the platform. 
+
+* SBI TIME 
+** This extension is mandatory only if SSTC ISA extension is not available.
+* SBI IPI
```

Comment:
Oh. I see the 'these two' language below. It may be better to just repeat the requirement for !IMSIC. Or it's ok just grouping them together. I don't have much of an opinion.

Reply:
Sure. Will do